doc/script_commands.txt

```diff
@@ -3197,6 +3197,70 @@ within the specified area - an x1/y1-x2/y2 square on the specified map.
 This is useful for maps that are split into many buildings, such as all the
 "*_in" maps, due to all the shops and houses.
 
+---------------------------------------
+
+*getunits(<type>{,<array_variable>})
+*getmapunits(<type>,<"map name">{,<array_variable>})
+*getareaunits(<type>,<"map name">,<x1>,<y1>,<x2>,<y2>{,<array_variable>})
+
+'getunits' function will return the number of objects of the type passed currently whole server.
+
+'getmapunits' function will return the number of objects of the type passed currently located
+on the specified map.
+
+'getareaunits' function will return the number of objects of the type passed currently located
+within the specified area - an x1/y1-x2/y2 square on the specified map.
+
+Type is the type of object to search for:
+
+	UNITTYPE_PC   - Character object
+	UNITTYPE_NPC  - NPC object
+	UNITTYPE_MOB  - Monster object
+	UNITTYPE_PET  - Pet object
+	UNITTYPE_HOM  - Homunculus object
+	UNITTYPE_MER  - Mercenary object
+	UNITTYPE_ELEM - Elemental object
```

Comment:
You could switch this type now. I would eventually like to drop this unit type since now we support `BL_` since the mapflag update. I think we are just missing `BL_NPC` from the export though.